packages/snowpack/package.json

```diff
@@ -15,6 +15,7 @@
   "scripts": {
     "build": "pika build"
   },
+  "bin": "pkg/dist-node/index.bin.js",
```

Comment:
this is annoying, but it's a better short-term fix than having "snowpack" marked as a workspace dependency and not a member of the workspace. There may be some way to get the `pkg` directory linked into the top-level node_modules, but I couldn't find it.